LeaderboardBackend.Jobs/Jobs/ScaffoldLeaderboardAndMod.cs

```diff
@@ -0,0 +1,55 @@
+using LeaderboardBackend.Jobs.Core;
+using LeaderboardBackend.Models.Entities;
+using BCryptNet = BCrypt.Net.BCrypt;
+
+namespace LeaderboardBackend.Jobs.Jobs;
+
+internal class ScaffoldLeaderboardAndMod : IJob
+{
+	private ApplicationContext Context;
+
+	public ScaffoldLeaderboardAndMod(ApplicationContext context)
+	{
+		Context = context;
+	}
+
+	public string CommandName => "modsetup";
+	public string Description => "Create a new leaderboard and a new user to mod it.";
+	public override string ToString() => $"{CommandName}: {Description}";
+
+	async Task IJob.Run()
+	{
+		Console.WriteLine("Please provide new leaderboard info.");
+		string name = Options.StringLine("Leaderboard Name");
+		string slug = Options.StringLine("Leaderboard Slug", validator: _ => true);
+		Leaderboard newLeaderboard = new()
+		{
+			Name = name,
+			Slug = slug,
+		};
+		Context.Leaderboards.Add(newLeaderboard);
+		await Context.SaveChangesAsync();
+		long? newLeaderboardId = Context.Leaderboards.Where(l => l.Name == name).FirstOrDefault()?.Id;
+		if (newLeaderboardId is null)
+		{
+			throw new Exception("Unexpected error creating leaderboard.");
```

Comment:
You know if - assuming this happens - that the DB row would still exist? I'm wondering if we'd need to worry about having to clean invalid rows if this ever happens. Same goes for L44, for creating users.

In fact; _would_ this ever even happen? Cuz if so, we should also add this check in `ScaffoldUser.Run()` too

Reply:
This wouldn't happen I'm just appeasing null reference type warnings

Reply:
Mmm